### ISSUE TITLE
Accessibility - Steps Screens - Bread Crumbs Not Announced to VoiceOver Users

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#36]
The bread crumbs are not being announced/read to VoiceOver users. VoiceOver skips right over the bread crumb links when they should be read as links. The text not being rad is "Home" and "Steps".

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
VoiceOver is now reading the bread crumbs links on the pages I tested.
